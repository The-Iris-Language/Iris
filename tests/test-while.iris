.. Tests simple while loop
class Main () {
    public: 
        int univ main() {
            int i = 0;
            while (i < 3) {
                i++;
                Olympus.println("in while loop");
            }
        }
        int getOne() {
            return 1;
        }
}