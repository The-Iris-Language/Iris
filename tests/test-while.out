in while loop
in while loop
in while loop
